script.js

```diff
@@ -0,0 +1,50 @@
+let startBtn = document.querySelector("#startButton");
+startBtn.addEventListener("click", playGame);
+//ash says we can use this as a global object
+let gameState = ["", "", "", "", "", "", "", "", ""];
+
+function decideTurn(currentPlayer) {
+    if (currentPlayer === 'o') {
+        return 'x';
+    }   else  {
+        return  'o';
+    }
+}
```

Comment:
Can lose this else as well since you're returning from the if
```js
function decideTurn(currentPlayer) {
    if (currentPlayer === 'o') {
        return 'x';
    } 
    return  'o';
}
```

This has the same end result